Gulpfile.js

```diff
@@ -172,6 +172,6 @@ gulp.task('watch', gulp.series(setDevelopment, function () {
         config.imgSrc + '/**/*',
         config.jsSrc + '/**/*.js',
         config.jsSrc + '/*.js'],
-        { interval: 1000 },
+        { interval: 1000, usePolling: true },
```

Comment:
Not sure what this is? Is this wanted?

Reply:
It lets you run Gulp in the background and automatically rebuild whenever you change a JS or CSS file. It's potentially useful when doing a bunch of small frontend changes, but I don't imagine it'd be used often. It doesn't cost us anything to have the option.